lib/src/kvs/tx.rs

```diff
@@ -394,6 +399,120 @@ impl Transaction {
 		}
 	}
 
+	/// Obtain a new change timestamp for a key
+	/// which is replaced with the current timestamp when the transaction is committed.
+	/// NOTE: This should be called when composing the change feed entries for this transaction,
+	/// which should be done immediately before the transaction commit.
+	/// That is to keep other transactions commit delay(pessimistic) or conflict(optimistic) as less as possible.
+	#[allow(unused)]
+	pub async fn get_timestamp<K>(&mut self, key: K, lock: bool) -> Result<Versionstamp, Error>
+	where
+		K: Into<Key>,
+	{
+		#[cfg(debug_assertions)]
+		trace!(target: LOG, "Get Timestamp");
```

Comment:
is it worth adding the Key in the trace message?

Reply:
Yeah I believe so! Added. Thanks for the suggestion ☺️